### PR TITLE
Fixes names on pretty much everything that needed fixing

### LINKED ISSUE #3561: Oxygen tank item names are fucked
## Description

**Screenshots**

image

**Additional context**

### PR DESCRIPTION
**Pull request by SweptWasTaken**
## About the PR 

Read the commits for changelog. I've endangered this PR being merged by removing the entity localization stuff but I think it's good considering we discussed it heavily on Discord and i'm pretty sure the resolution was "No lol entity localization would be too much". (I'm of that opinion anyway).

Most of these things were broken due to entity localization being dummy broken and people trying to follow that standard.

I've also updated a ton of naming for entities that didn't fit the capitalization standard, and tried to make circuit boards more legible. 

This PR closes #3561 and  #3633.

**Changelog**

:cl: Swept
- fix: Handheld tanks and solar stuff now has correct naming
- fix: Fixed some other naming related stuff

### CODE REVIEW

**Comment by RemieRichards** [comment]
As has since been discussed on discord:
> we discussed it heavily on Discord and i'm pretty sure the resolution was "No lol entity localization would be too much".

Never happened.

**Comment by SweptWasTaken** [comment]
> 
> 
> As has since been discussed on discord:
> 
> > we discussed it heavily on Discord and i'm pretty sure the resolution was "No lol entity localization would be too much".
> 
> Never happened.

I misremembered actually was a conversation about it being broken with serv3.

**Comment by Peptide90** [comment]
> > As has since been discussed on discord:
> > > we discussed it heavily on Discord and i'm pretty sure the resolution was "No lol entity localization would be too much".
> > 
> > 
> > Never happened.
> 
> I misremembered actually was a conversation about it being broken with serv3.

So does this mean that removing it isn't the way to go?  Seems the foundations are there it just needs updated for serv3

**Comment by SweptWasTaken** [comment]
> 
> 
> > > As has since been discussed on discord:
> > > > we discussed it heavily on Discord and i'm pretty sure the resolution was "No lol entity localization would be too much".
> > > 
> > > 
> > > Never happened.
> > 
> > 
> > I misremembered actually was a conversation about it being broken with serv3.
> 
> So does this mean that removing it isn't the way to go? Seems the foundations are there it just needs updated for serv3

No because it's broken right this second and that fix has no ETA. Plus, working with ftl files and the yaml without any GUI is CBT so I think it's best to wait for entity localization to get more tooling for the future.

**Comment by SweptWasTaken** [comment]
Let me know when someone reviews + wants to merge this and i'll speed-fix the conflicts, i'm just letting it sit for now since i'd rather solve conflicts in one batch.

File: Resources/Prototypes/Entities/Objects/Power/antimatter_part.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 - type: entity
   id: AMEPart
-  name: Antimatter Engine Part
+  name: AME engine part
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Objects/Power/antimatter_part.yml:3]: antimatter engine engine?

**Comment by SweptWasTaken** [comment]
Just a mess. Will re-PR.

**Pull request closed without merging**